src/shout_subgroup/list_subgroup.py

```diff
@@ -6,12 +6,44 @@
 from telegram.ext import ContextTypes
 
 from shout_subgroup.database import session
-from shout_subgroup.exceptions import NotGroupChatError
-from shout_subgroup.models import SubgroupModel
-from shout_subgroup.repository import find_all_subgroups_in_group_chat
+from shout_subgroup.exceptions import NotGroupChatError, SubGroupDoesNotExistsError
+from shout_subgroup.models import SubgroupModel, UserModel
+from shout_subgroup.repository import (find_all_subgroups_in_group_chat,
+                                       find_subgroup_by_telegram_group_chat_id_and_subgroup_name,
+                                       find_all_users_in_subgroup)
 from shout_subgroup.utils import is_group_chat
 
 
+async def _handle_list_subgroups(update: Update, db: Session, telegram_group_chat_id: int):
```

Comment:
Refactor: Moved this logic into it's own function b/c it makes the handler function succinct.